app/components/markdown/markdown.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class="<%= class_attr %> rich-content">
+<div <%= attrs %>>
```

Comment:
This applies a class and still allows you to add more classes when you render the component.

```suggestion
<div <%= attrs.merge(class: 'rich-content') %>>
```

Take a look at the specs here to see what other methods are available: https://github.com/tactilenews/100eyes/blob/main/spec/components/attribute_bag_spec.rb

Reply:
Cool, thanks :green_heart: